<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="inputLeft">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputRight">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAdd">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRemove">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>541</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>displayLabel</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbEn2se">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>98</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>EN 2 SE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbSe2en">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>98</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>SE 2 EN</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
